src/responses/handleAddExpense/handleAddExpense.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-var */
+'use strict';
+
+const { ERROR } = require('../../errors');
+const { getExpenses } = require('./getExpenses');
+const { getNewExpense } = require('./getNewExpense');
+const { saveExpenses } = require('./saveExpenses');
+const { getValidatedExpense } = require('./getValidatedExpense');
+
+async function handleAddExpense(request, response) {
+  try {
+    var expenses = await getExpenses();
```

Comment:
Avoid using vars. Consider rewriting a logic with new js features here

Reply:
But what is the problem with var in this case?  It is not deprecated, and its functional scope only makes the code more readable and elegant. Without var this code will look terrible and will be work slowly

Reply:
Its functional scope can become a problem in case the function will be extended